src/veh_interact.cpp

```diff
@@ -1838,7 +1833,7 @@ void veh_interact::display_details( const vpart_info *part )
                    "%s: <color_ltgray>%.1f%s</color>",
                    small_mode ? _("Wgt") : _("Weight"),
                    convert_weight(item::find_type( part->item )->weight),
-                   OPTIONS["USE_METRIC_WEIGHTS"].getValue() == "lbs" ? "lb" : "kg");
```

Comment:
Someone forgot to make units translatable here, so I decided to solve this issue radically.